packages/terra-form-select/src/multiple/Frame.jsx

```diff
@@ -223,7 +228,7 @@ class Frame extends React.Component {
             </li>
           ) : null}
         <li className={cx('search-wrapper')}>
-          <input {...inputAttrs} value={searchValue} />
+          {(this.props.isSearchDisabled) ? <div className={cx('placeholder')}>{placeholder || '\xa0'}</div> : <input {...inputAttrs} value={searchValue} /> }
```

Comment:
In a case when a value is already selected up-front, is this by design that we'd want to show the placeholder _only_ when search is disabled?

Before: 

![Screen Shot 2019-11-12 at 2 49 46 PM](https://user-images.githubusercontent.com/10090859/68658625-d0b66a80-055b-11ea-8ad7-20c227059403.png)

After: 
![Screen Shot 2019-11-12 at 2 50 00 PM](https://user-images.githubusercontent.com/10090859/68658658-dd3ac300-055b-11ea-9efa-c170c23cebaa.png)

Reply:
replacing div with input fixed this : 148aaf18db9acf04ced1b29d35d98c3ada852ee4